test/e2e/util.go

```diff
@@ -275,7 +275,7 @@ func createSkipperPodSpec(route string, port int32) corev1.PodSpec {
 		Containers: []corev1.Container{
 			{
 				Name:  "skipper",
-				Image: "container-registry.zalando.net/teapot/skipper:v0.15.23",
+				Image: "container-registry.zalando.net/teapot/skipper:latest",
```

Comment:
It makes sense to use `latest` tag here.

Reply:
We can't use `latest` in e2e because we check that the images are production ready as part of e2e.

Reply:
then that's the problem I think, will push and see. Thanks!